src/Fantomas.Tests/DotIndexedGetTests.fs

```diff
@@ -70,3 +70,27 @@ myList.[7]
         otherArgument4
     )
 """
+
+[<Test>]
+let ``should not merge tokens inside parentheses, 1407`` () =
+    formatSourceString
+        false
+        """
+let inline (=??) x = (=!) x
+let mySampleMethod() =
+    let result = Ok {| Results = [] |}
+    (Result.okValue result).Results.[0] |> Result.isOk =?? true
+"""
+        config
```

Comment:
Please update the config, the problem was reported using:
```fsharp
    { config with
                SpaceBeforeLowercaseInvocation = false
                SpaceBeforeColon = true
                MaxIfThenElseShortWidth = 25
                MultilineBlockBracketsOnSameColumn = true
                KeepIfThenInSameLine = true
                AlignFunctionSignatureToIndentation = true
                AlternativeLongMemberDefinitions = true
                MultiLineLambdaClosingNewline = true }
```

the bug occurred using this combination of settings.